README.md

```diff
@@ -1,151 +1,80 @@
-## About
```

Comment:
This is the readme of the conda-forge repo. Please do not change anything in here.

Reply:
My bad I unintentionally mistyped.